css/terminal.css

```diff
@@ -0,0 +1,290 @@
+@keyframes blink-animation {
```

Comment:
Is this file a mostly unchanged copy of upstream? If so I'd rather it lived in css/vendor/terminal.css

Reply:
I removed a few lines that's not used and added term-fgx8 to term-fgx15, the rest is not changed, but we might want to customize it further by choosing different color based on dark vs light theme.